docs/orchestrating-elastic-stack-applications/apm-server.asciidoc

```diff
@@ -75,6 +75,28 @@ apm-server-quickstart-apm-server-69b447ddc5-fflc6   1/1     Running   0
 [id="{p}-apm-advanced-configuration"]
 == Advanced configuration
 
+[id="{p}-apm-agent-central-configuration"]
+=== Use APM Agent central configuration
```

Comment:
Should we mention that this feature is only available after `7.5.1` ? [I had some issues with 7.5.0](https://github.com/elastic/cloud-on-k8s/pull/3155#issuecomment-637299108) when I did some tests.